packages/jest-config/src/normalize.js

```diff
@@ -354,6 +354,9 @@ function normalize(config: InitialConfig, argv: Object = {}) {
       case 'persistModuleRegistryBetweenSpecs':
       case 'preset':
       case 'replname':
+      case 'reporters':
+        value = _replaceRootDirTags(config.rootDir, config[key]);
```

Comment:
It is necessary, `jest-validate` only checks for your config to be valid against an example and throws otherwise, it doesn't modify anything.